saleor/shipping/utils.py

```diff
@@ -44,4 +46,5 @@ def convert_to_shipping_method_data(
         minimum_delivery_days=shipping_method.minimum_delivery_days,
         metadata=shipping_method.metadata,
         private_metadata=shipping_method.private_metadata,
+        minimum_order_price=getattr(listing, "minimum_order_price", None),
```

Comment:
Why getattr? Isn't it a regular model field?

Reply:
It's because `listing` is optional here but I can move the assignment to the condition check above if you prefer it that way